lib/imgix/client.rb

```diff
@@ -62,6 +62,9 @@ def purge(path)
     end
 
     def prefix(path)
+      msg = "Warning: `Client::prefix' will take zero arguments " \
+        "in the next major version.\n"
+      warn msg
```

Comment:
What if we only throw the warning if an argument `path` if passed in? No need to emit a warning for something a user isn't actually doing.